dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -211,7 +215,9 @@
 
             <SolidColorBrush x:Key="NavigationViewSelectionIndicatorForeground" Color="{ThemeResource SystemColorHighlightTextColor}" />
 
-            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
+            <StaticResource x:Key="NavigationViewContentGridBorderBrush" ResourceKey="SubtleFillColorTransparentBrush" />
```

Comment:
it's HC, are we supposed to user other brushes than **System***?:)

Reply:
All the NavViewItemBorderBrush for HC is using the same brush. But you're right, maybe it should be `SystemControlTransparentBrush` for consistency